### DISCUSSION TITLE
TP5 - ¿Como se documentan los struct?

### DISCUSSION THREAD

**Discussion by martinjguevara** [INGCOM-UNRN-P1/cursada-2023#170] [Preguntas y respuestas] [answered]
Como seria la documentación de una función con parámetros de tipo estructura? Se detalla en las pre-condiciones o en los  @param?

**Comment by martinvilu** [comment, marked as answer]
Lo mismo que los argumentos tradicionales, indicando que valores puntuales son retornados y que valores son importantes al entrar.

No hay un formato específico.

Si _por ejemplo_ hacen la suma lenta con un argumento para el `dividendo` y `divisor`, indicar que el valor `divisor` de la estructura no puede ser cero, ya que la división no es posible.
